src/components/badges/index.ts

```diff
@@ -0,0 +1,5 @@
+import Badges from './Badges';
+
+export * from './Badges';
```

Comment:
we will export props and types from a different file in the future, but for now this is ok :D